src/main/java/uk/nhs/hee/tis/trainee/sync/service/TcsSyncService.java

```diff
@@ -54,7 +54,7 @@
   private static final String TABLE_PROGRAMME_MEMBERSHIP = "ProgrammeMembership";
   private static final String TABLE_CURRICULUM = "Curriculum";
 
-  private static final Map<String, String> TABLE_NAME_TO_API_PATH = Map.of(
+  static final Map<String, String> TABLE_NAME_TO_API_PATH = Map.of(
```

Comment:
I can't see any use outside of this class.  Do we need to make it package scope?

Reply:
Whoops, I think I was originally playing around with using it for the test. Rolled that change back.